### PR TITLE
Added range service(#68)

### LINKED ISSUE #68: Implement Range Service
- create
- delete
- update

### PR DESCRIPTION
**Pull request by AndriiKret**

### CODE REVIEW

File: src/app/services/range.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+class RangeService:
+    """
+    Class of range service
+    """
+
+    @staticmethod
+    @transaction_decorator
+    def create(range_min, range_max):
+        """
+        Creates Range object and saves if not created, or if created returns existing object
```
Comment (yurdosii) [inline src/app/services/range.py:19]: New line

File: src/app/services/range.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        instance = Range(min=range_min, max=range_max)
+        DB.session.add(instance)
+        return instance
+
+    @staticmethod
+    @transaction_decorator
+    def update(range_id, range_min=None, range_max=None):
+        """
+        Updates object with id
+        :param range_id:
```
Comment (yurdosii) [inline src/app/services/range.py]: New line

File: src/app/services/range.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            instance.max = range_max
+        DB.session.merge(instance)
+        return instance
+
+    @staticmethod
+    @transaction_decorator
+    def delete(range_id):
+        """
+        Delete object with id param if exists
+        :param range_id: int
```
Comment (yurdosii) [inline src/app/services/range.py]: New line

File: src/app/services/range.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        if instance is None:
+            raise RangeNotExist()
+        DB.session.delete(instance)
+        return True
+
+    @staticmethod
+    def get_by_id(range_id):
+        """
+        Get Range model object by id
+        :param range_id: id of range
```
Comment (yurdosii) [inline src/app/services/range.py]: New line

File: src/app/services/range.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        :return: Range instance or None
+        """
+        instance = Range.query.get(range_id)
+        return instance
+
+    @staticmethod
+    def filter(range_id=None, range_min=None, range_max=None):
+        """
+        Filter method
+        :param range_id: range id
```
Comment (yurdosii) [inline src/app/services/range.py]: New line

File: src/app/services/range.py [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    @transaction_decorator
+    def update(range_id, range_min=None, range_max=None):
+        """
+        Updates object with id
+        :param range_id:
+        :param range_min: min value which can be changed
+        :param range_max: max value which can be changed
+        :return: object if changed, or None if object with id doesn't exist
+        """
+        instance = RangeService.get_by_id(range_id)
```
Comment (yurdosii) [inline src/app/services/range.py:43]: Use get_by_id instead

File: src/app/services/range.py [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+    @staticmethod
+    @transaction_decorator
+    def delete(range_id):
+        """
+        Delete object with id param if exists
+        :param range_id: int
+        :return: True after deletion of None if object with id param doesn't exist
+        """
+        instance = Range.query.get(range_id)
```
Comment (yurdosii) [inline src/app/services/range.py]: Use get_by_id instead

**Review by IvanKostiv — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Review by dtymkiv — APPROVED** [review]

**Pull request merged**